src/main/java/org/apache/sling/resourceresolver/impl/mapping/MapEntries.java

```diff
@@ -261,6 +269,10 @@ protected void initializeVanityPaths() throws IOException {
         }
 
     }
+    
+    protected int getTraversalTimer(){
```

Comment:
'timer' is maybe not the best term here. Delay? Interval?

Reply:
TraversalRetryInterval sounds good.. a bit mindless on picking name - will fix.